server/src/apiClient/YelpClient.js

```diff
@@ -0,0 +1,26 @@
+import got from "got"
+import config from "../config.js"
+
+
+const yelpApiKey = config.yelpApiKey.secret
+
+class YelpClient {
+  static async getPlaces() {
+    try {
+      const url = `https://api.yelp.com/v3/businesses/search?location=Boston`
```

Comment:
so that you can then use them in this url string, and add them add a query param
`https://api.yelp.com/v3/businesses/search?location=Boston&category=indian`